app/setup_script.sql

```diff
@@ -0,0 +1,51 @@
+-- This is the setup script that will run while installing your application instance in a consumer's account.
+-- For more information on how to create setup file, visit https://docs.snowflake.com/en/developer-guide/native-apps/creating-setup-script
+
+-- A general guideline to building this script looks like:
+
+
+-- this app exposes 2 application roles: admin and viewer
+create application role if not exists app_admin;
+create application role if not exists app_viewer;
+
+-- viewer permissions are a subset of those for the admin
+grant application role app_viewer to application role app_admin;
+
+-- the config data schema holds configuration data
+-- we create a non-versioned schema "if not exists"
+-- so that we do not clobber state whenever we upgrade the app
+create schema if not exists config_data;
+    grant usage on schema config_data to application role app_admin;
+    execute immediate from './config_data/configuration.sql';
+
+-- non-versioned schema to prevent data loss on upgrade
+-- we'll also store our task(s) here
+create schema if not exists warnings_data;
+    grant usage on schema warnings_data to application role app_admin;
+    execute immediate from './warnings_data/warnings_reading_cursor.sql';
+    execute immediate from './warnings_data/warnings.sql';
+
+-- the ui schema holds all streamlits
+create or alter versioned schema ui;
```

Comment:
Do we need alter here since it is a versioned schema? Maybe create or replace?
Same for versioned schemas on 36, 42 and 48.

Reply:
Yes, `create or alter` is the correct path for versioned schemas.